src/components/VtkObliqueView.vue

```diff
@@ -316,11 +316,32 @@ export default defineComponent({
     });
 
     const { baseImageRep } = useSceneBuilder<
-      vtkResliceRepresentationProxy
+      vtkObliqueRepresentationProxy
     >(viewID, {
       baseImage: curImageID
     });
 
+    const obliqueRep = baseImageRep?.value;
+    if (obliqueRep) {
+      switch(viewDirection.value) {
+        case 'Left':
+        case 'Right':
+          obliqueRep.setOutlineColor([1, 0, 0]);
+          break;
+        case 'Posterior':
+        case 'Anterior':
+          obliqueRep.setOutlineColor([0, 1, 0]);
+          break;
+        case 'Superior':
+        case 'Inferior':
+          obliqueRep.setOutlineColor([0, 0, 1]);
+          break;
+        default:
+          obliqueRep.setOutlineColor([0, 0, 0]);
+          break;
+      };
+    }
```

Comment:
I recommend putting this inside a `watch(baseImageRep, (obliqueRep) => ...), { immediate: true })`.